src/packages/frontend/frame-editors/x11-editor/xpra-client.ts

```diff
@@ -63,7 +63,7 @@ export class XpraClient extends EventEmitter {
   constructor(options: Options) {
     super();
     this.record_active = throttle(this.record_active.bind(this), 30000);
-    this.connect = reuseInFlight(this.connect);
+    this.connect = reuseInFlight(this.connect.bind(this));
```

Comment:
essential is this line, a missing `bind(this)` ... just like with the throttle in the line above.

other changes are just sorting and formatting